### ISSUE TITLE
Metrics tab on table page is broken

### ISSUE THREAD

**Issue by robertsami** [yugabyte/yugabyte-db#5582]
http://portal.dev.yugabyte.com/universes/4ea18f4b-3b1c-46ea-b291-aca81eba88a1/tables/000030a9-0000-3000-8000-000000004019?tab=metrics

```
Failed to load resource: the server responded with a status of 505 (HTTP Version Not Supported)
/api/v1/insecure_login:1 Failed to load resource: the server responded with a status of 401 (Unauthorized)
2VM49 react_devtools_backend.js:2273 TypeError: Cannot read property 'title' of undefined
    at a.value (GraphPanel.js:229)
    at Ba (2.b87f9cd0.chunk.js:sourcemap:1)
    at Pa (2.b87f9cd0.chunk.js:sourcemap:1)
    at gs (2.b87f9cd0.chunk.js:sourcemap:1)
    at cl (2.b87f9cd0.chunk.js:sourcemap:1)
    at sl (2.b87f9cd0.chunk.js:sourcemap:1)
    at Js (2.b87f9cd0.chunk.js:sourcemap:1)
    at 2.b87f9cd0.chunk.js:sourcemap:1
    at t.unstable_runWithPriority (2.b87f9cd0.chunk.js:sourcemap:1)
    at $i (2.b87f9cd0.chunk.js:sourcemap:1)
overrideMethod @ VM49 react_devtools_backend.js:2273
2.b87f9cd0.chunk.js:sourcemap:1 Uncaught TypeError: Cannot read property 'title' of undefined
    at a.value (GraphPanel.js:229)
    at Ba (2.b87f9cd0.chunk.js:sourcemap:1)
    at Pa (2.b87f9cd0.chunk.js:sourcemap:1)
    at gs (2.b87f9cd0.chunk.js:sourcemap:1)
    at cl (2.b87f9cd0.chunk.js:sourcemap:1)
    at sl (2.b87f9cd0.chunk.js:sourcemap:1)
    at Js (2.b87f9cd0.chunk.js:sourcemap:1)
    at 2.b87f9cd0.chunk.js:sourcemap:1
    at t.unstable_runWithPriority (2.b87f9cd0.chunk.js:sourcemap:1)
    at $i (2.b87f9cd0.chunk.js:sourcemap:1)
ui_theme:1 Failed to load resource: the server responded with a status of 505 (HTTP Version Not Supported)
```

**Comment by iSignal** [comment]
Also, not exactly related to the metrics tab - but another UI issue on the tables section - 

The alignment of values on the main tables list is broken. On wide screens, the values do not follow the headers. 
![Screen Shot 2020-09-15 at 7 01 51 PM](https://user-images.githubusercontent.com/8207010/93370129-33743480-f805-11ea-9e14-f9897d98a630.png)

**Comment by sshev** [comment]
Part of the configuration of the metric graphs was unintentionally removed at https://github.com/yugabyte/yugabyte-db/commit/efc3d41ae85239d9685cad24dcbb98cacb1caf15#diff-2c3a1d35efea16b272279b22281a36f9L117-L145 which caused page malfunctioning

**Comment by sshev** [comment]
Fixed in `master` and backported to `2.1` and `2.2` branches
